### PR TITLE
Github actions nodejs add machines and non-English tests

### PR DESCRIPTION
**Pull request by iliadrm**

### CODE REVIEW

File: binding/nodejs/index.test.js [hunk 1]
```diff
@@ -158,6 +203,122 @@ describe("manual paths", () => {
...
+describe("keyword detection in DE", () => {
+  test("single keyword single detection", () => {
+    let porcupineEngine = new Porcupine(
+      ACCESS_KEY,  
+      keywordPathsSingleDe,
+      SENSITIVITIES_1,
+      MODEL_PATH_DE
+    );
+
+    let counts = porcupineDetectionCounts(
```
Comment (ksyeo1010) [inline binding/nodejs/index.test.js:215]: for the `porcupineDetectionCounts`, we should do the same as in python with paths:

```javascript
const path = require("path")

let some_path = path.join(__dirname, "path_relative_to_this_directory")
```

**Review by ksyeo1010 — APPROVED** [review]

File: binding/nodejs/index.test.js [hunk 2]
```diff
@@ -158,6 +205,122 @@ describe("manual paths", () => {
...
+      porcupineEngine,
+      WAV_PATH_SINGLE_KEYWORD_FR
+    );
+    let monChouchouCount = counts.get(0);
+    expect(monChouchouCount).toEqual(1);
+  });
+
+  test("multiple keyword multiple detection", () => {
```
Comment (laves) [inline binding/nodejs/index.test.js:303]:
```suggestion
  test("multiple keyword multiple detection in FR", () => {
```

File: binding/nodejs/index.test.js [hunk 3]
```diff
@@ -158,6 +205,122 @@ describe("manual paths", () => {
...
+
+    let counts = porcupineDetectionCounts(
+      porcupineEngine,
+      WAV_PATH_SINGLE_KEYWORD_ES
+    );
+    let manzanaCount = counts.get(0);
+    expect(manzanaCount).toEqual(1);
+  });
+
+  test("multiple keyword multiple detection", () => {
```
Comment (laves) [inline binding/nodejs/index.test.js:265]:
```suggestion
  test("multiple keyword multiple detection in ES", () => {
```

File: binding/nodejs/index.test.js [hunk 4]
```diff
@@ -158,6 +205,122 @@ describe("manual paths", () => {
...
+
+    let counts = porcupineDetectionCounts(
+      porcupineEngine,
+      WAV_PATH_SINGLE_KEYWORD_DE
+    );
+    let heuschreckeCount = counts.get(0);
+    expect(heuschreckeCount).toEqual(1);
+  });
+
+  test("multiple keyword multiple detection", () => {
```
Comment (laves) [inline binding/nodejs/index.test.js:225]:
```suggestion
  test("multiple keyword multiple detection in DE", () => {
```

**Review by laves — CHANGES_REQUESTED** [review]

File: binding/nodejs/index.test.js [hunk 5]
```diff
@@ -26,9 +26,6 @@ const MODEL_PATH_ES = "../../lib/common/porcupine_params_es.pv";
 const MODEL_PATH_FR = "../../lib/common/porcupine_params_fr.pv";
 
 const SENSITIVITIES_1 = [0.5];
```
Comment (laves) [inline binding/nodejs/index.test.js:28]: can you just rename this like `DEFAULT_SENSITIVITY_ARRAY` or something? since we're not going to create anymore.

**Review by laves — CHANGES_REQUESTED** [review]

**Pull request merged**